tests/tensorflow/r2.9/common.libsonnet

```diff
@@ -76,7 +73,9 @@ local mixins = import 'templates/mixins.libsonnet';
   RunNightly:: {
     schedule: functional_schedule,
   },
+  // Running functional tests at Midnight PST Daily.
```

Comment:
Do you want to run convergence tests every day?

Reply:
Yes. For now. We are already late this release cycle. Tensorflow is close to their release, and we haven't tested even RC0 yet. So want to start testing daily. Once we get enough confidence, we will reduce the frequency of the tests.